db/migrate/20230807033021_add_group_to_web_hook_event_type.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddGroupToWebHookEventType < ActiveRecord::Migration[7.0]
+  def change
+    add_column :web_hook_event_types, :group, :string
```

Comment:
Personally I think this should be an `:integer` and then have an Enum for the different groups in the model, then we are not repeating string data tons of times in the DB. Then you could have `WebHookEventType.groups[:topic]`